app/controllers/api/payments/subscriptions_controller.rb

```diff
@@ -41,4 +41,16 @@ def update_plan
       }
     }
   end
+
+  def create_paypal_premium
+    subscription = ::Payments::Paypal::Subscription::CreateForPremium.(current_user, params[:interval])
```

Comment:
What should be do if no data is returned? That would mean there is a paypal error somehow.